api/src/opentrons/protocol_engine/commands/load_liquid.py

```diff
@@ -0,0 +1,62 @@
+"""Load liquid command request, result, and implementation models."""
+from pydantic import BaseModel, Field
+from typing import Optional, Type, Dict
+from typing_extensions import Literal
+
+from .command import AbstractCommandImpl, BaseCommand, BaseCommandCreate
+
+
+LoadLiquidCommandType = Literal["loadLiquid"]
+
+
+class LoadLiquidParams(BaseModel):
+    """Payload required to load a liquid into a well."""
+
+    liquidId: str = Field(
+        ...,
+        description="Unique identifier generated by the app of instance of liquid to load.",
+    )
+    labwareId: str = Field(
+        ...,
+        description="Unique identifier of labware to load liquid into.",
+    )
+    volumeByWell: Dict[str, int] = Field(
```

Comment:
Might rename this here and in JSONv6 to match existing conventions. Also needs to accept a `float`, not an `int`

```suggestion
    volumeByWellName: Dict[str, float] = Field(
```

It's important to be very explicit about what an object's keys represent

Reply:
should I change this in the 6.json as well? @mcous

Reply:
if we change to `volumeByWellName` there are several places in the frontend where we'd need to also change. not saying we shouldnt do that, just saying it's gotta be a full stack change or stuff will break

Reply:
Will any _production_ stuff break? I think we tried to communicate pretty explicitly that the backend didn't want to consider the v6 loadLiquid schema baked until we had a chance to get the backend implementation in place

I worry that this field name is confusing if/when we start assigning unique identifiers to wells and the key structure for this dict becomes a lot more ambiguous

Reply:
synced with @mcous irl, we agreed not to change name now in favor of doing more changes later. we don't wanna change schema v6 across releases